src/components/FileUpload.js

```diff
@@ -0,0 +1,40 @@
+const Data = (props) => {
+    const data = props.data;
+    const getTotalPassed = () => {
+      return data.filter((item) => item.status === "PASSED").length;
+    }
+  
+    const getTotalFailed = () => {
+      return data.filter((item) => item.status === "FAILED").length;
+    }
+  
+    const getRemaining = () => {
+      return data.filter((item) => !item.status).length;
+    }
```

Comment:
**Suggestion/nit (non-blocking):** If you like it better stylistically, you can actually just set these as values right here (if you don't want to have inline function calls)...

```typescript
const totalPassed = data.filter((item) => item.status === "PASSED").length;

// ... later
{ totalPassed }
```

The values will be recalculated on every rerender, and the component will rerender whenever the props change 👍

Reply:
> you can actually just set these as values right here

Ah, yes! This is preferable. Fixed in [413926e](https://github.com/Automattic/compare-snapshot-tool/pull/5/commits/413926eb38c598e77fb80818ec50e130846dff63).